src/syft/grid/duet/signaling_client.py

```diff
@@ -22,6 +22,9 @@ def __init__(
         # Use Signaling Server metadata
         # to build client route
         conn = conn_type(url=url)  # type: ignore
+
+        print(conn._get_metadata())
```

Comment:
MAYBE: Use debug?

Reply:
Good spot, I think thats an accidental left over print statement from when we were testing the change from json to protobuf for metadata, so i will just remove it.